theme/static/css/hydroshare_core.css

```diff
@@ -399,6 +399,13 @@ a.sign-out-inline:hover {
 	overflow: hidden;
 }
 
+#add-author-modal .author-types > button {
+	flex-grow: 1;
+	flex-basis: 0;
+	text-overflow: ellipsis;
+	overflow: hidden;
+}
+
 /* warning access removal dialog title */
```

Comment:
does this belong in hydroshare_core.css or a smaller, more specific area?

Reply:
Looks like the other author stuff is already in there, so all set...

Reply:
Correct. Right now there is not a more specific stylesheet to put this in, but that could change in the future if we decide to optimize this file.